src/Core/Grid/Definition/Factory/TitleGridDefinitionFactory.php

```diff
@@ -0,0 +1,237 @@
+                    ->setOptions([
+                        'field' => 'type',
+                        'sortable' => false,
+                    ])
+            )
+            ->add(
+                (new ImageColumn('image'))
+                    ->setName($this->trans('Image', [], 'Admin.Global'))
+                    ->setOptions([
+                        'src_field' => 'image',
+                    ])
+            )
+            ->add(
+                (new ActionColumn('actions'))
+                    ->setName($this->trans('Actions', [], 'Admin.Global'))
+                    ->setOptions([
+                        'actions' => (new RowActionCollection())
+                            ->add(
+                                (new LinkRowAction('edit'))
+                                    ->setName($this->trans('Edit', [], 'Admin.Actions'))
+                                    ->setIcon('edit')
+                                    ->setOptions([
+                                        'route' => 'admin_title_edit',
+                                        'route_param_name' => 'titleId',
+                                        'route_param_field' => 'id_gender',
+                                        'clickable_row' => true,
+                                    ])
+                            )
+                            ->add(
+                                $this->buildDeleteAction(
+                                    'admin_title_delete',
+                                    'titleId',
+                                    'id_gender',
+                                    Request::METHOD_DELETE
+                                )
+                            ),
+                    ])
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getFilters()
+    {
+        return (new FilterCollection())
+            ->add(
+                 (new Filter('id_gender', NumberType::class))
+                     ->setTypeOptions([
+                         'required' => false,
+                         'attr' => [
+                             'placeholder' => $this->translator->trans('Search ID', [], 'Admin.Actions'),
+                         ],
+                     ])
+                     ->setAssociatedColumn('id_gender')
+             )
+            ->add(
+                 (new Filter('name', TextType::class))
+                     ->setTypeOptions([
+                         'required' => false,
+                         'attr' => [
+                             'placeholder' => $this->translator->trans('Search title', [], 'Admin.Shopparameters.Feature'),
+                         ],
+                     ])
+                     ->setAssociatedColumn('name')
+             )
+            ->add(
+                 (new Filter('type', ChoiceType::class))
+                     ->setTypeOptions([
+                         'choices' => $this->genderChoiceProvider->getChoices(),
```

Comment:
```suggestion
                         'choices' => $this->genderChoiceProvider->getChoices(),
                         'placeholder' => '--',
```

You can set the placeholder here, so that the gender provider doesn't need to handle the empty value internally.